radio/src/gui/colorlcd/menus.h

```diff
@@ -54,6 +54,8 @@ enum MenuIcons {
   ICON_MODEL_SPECIAL_FUNCTIONS,
   ICON_MODEL_LUA_SCRIPTS,
   ICON_MODEL_TELEMETRY,
+  ICON_MODELSELECT,
+  ICON_MODELSELECT_CATEGORY,
```

Comment:
I'd like to buy a `_` to make it look like the others.

Reply:
it's actually the reason I did not add the _
their general pattern is ICON MENU CATEGORY
MODEL_SELECT, would it be the category select in the model menu, or not?

Reply:
Just change it to `ICON_MODEL_SELECT` & `ICON_MODEL_SELECT_CATEGORY`.